op-test

```diff
@@ -633,6 +633,16 @@ class CrashSuite():
     def suite(self):
         return PowerNVDump.crash_suite()
 
+class OSdumpSuite():
+    '''Crash Test Suite for IO'''
```

Comment:
suggest  can we have this Doc String as generic as like 
''' Crash test Suite which Verify operating System Crash Functionality '''